proposals/lip-0043.md

```diff
@@ -471,20 +487,25 @@ def execute(trs: Transaction) -> None:
                         }
                     }
                 }
-            }
+            },
+            "mainchainCertificateThreshold": {
+                "dataType": "uint64",
+                "fieldNumber": 4
+            },
         }
     }
 
     message = encode(registrationSignatureMessageSchema,
         {
-        "ownChainID": trs.params.ownChainID,
-        "ownName": trs.params.ownName,
-        "mainchainValidators": trs.params.mainchainValidators
+            "ownChainID": trs.params.ownChainID,
+            "ownName": trs.params.ownName,
+            "mainchainValidators": trs.params.mainchainValidators,
+            "mainchainCertificateThreshold": trs.params.mainchainCertificateThreshold
         }
     )
 
     # verifyWeightedAggSig is specified in LIP 0062.
-    if verifyWeightedAggSig(blsKeys, MESSAGE_TAG_CHAIN_REG, trs.params.ownChainID, trs.params.aggregationBits, trs.params.signature, bftWeights, certificateThreshold, message) == False:
+    if verifyWeightedAggSig(blsKeys, trs.params.aggregationBits, trs.params.signature, MESSAGE_TAG_CHAIN_REG, trs.params.ownChainID, bftWeights, certificateThreshold, message) == False:
```

Comment:
perhaps break this line for better readability.

Reply:
done